rocAL/rocAL/include/pipeline/tensor.h

```diff
@@ -226,6 +232,7 @@ class rocalTensor {
     unsigned copy_data(hipStream_t stream, void* host_memory, bool sync);
 #endif
     unsigned copy_data(void* user_buffer);
+    unsigned copy_data(void* user_buffer, uint max_x1, uint max_y1);
```

Comment:
Please check if is better to change max_x1, max_y1 to max_width, max_height

Reply:
Its for audio use case , i think width and height would not be correct for Audio usecases. If you think theres a better name, please suggest